src/bioregistry/data/bioregistry.json

```diff
@@ -79660,7 +79663,7 @@
       "prefix": "omia",
       "uri_format": "http://omia.angis.org.au/retrieve.shtml?pid=$1"
     },
-    "uri_format": "https://omia.org/OMIA$1"
+    "uri_format": "https://omia.org/phene/omia:$1/"
```

Comment:
@marius-mather it will create a huge difficulty for us if you add this trailing slash.. can we avoid it? We try to be able to replace prefixes simply, but with a trailing slash we need to add extra logic.. thanks for considering..

Reply:
Ah yes, sorry, I think you had mentioned this previously. I think it should be fine but let me do some testing on my end.